package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "eslint packages/ --ext .ts,.tsx .",
     "test": "yarn workspace @koh/server test",
     "test:cov": "yarn workspace @koh/server test:cov",
+    "test:update-snapshots": "yarn workspace @koh/server test:update-snapshots",
```

Comment:
i always had trouble getting the snapshots to update properly and couldn't figure out a command for it, so if this works then thank you :)

Reply:
it works as far as i can tell, it might have issue with running the integration snapshots, but I added a quick command for that in the server package.json